src/zzz_od/application/coffee/coffee_app.py

```
@@ -63,19 +64,27 @@ def __init__(self, ctx: ZContext):
         注意初始化要全面 方便一个指令重复使用
         """
-        pass
+        self.turn_compensator.reset()
+        self.retried_transport: bool = False
 
+    @node_from(from_name='等待咖啡店加载', success=False)
     @operation_node(name='传送', is_start_node=True)
     def transport(self) -> OperationRoundResult:
+        if self.previous_node.name == '等待咖啡店加载':
+            if self.retried_transport:
+                return self.round_fail(status='等待咖啡店加载失败，重传送超限')
```

Ruff (0.15.13)

[warning] String contains ambiguous `，` (FULLWIDTH COMMA). Did you mean `,` (COMMA)?

(RUF001)